projects/angular/components/ui-grid/src/ui-grid.component.spec.ts

```diff
@@ -2284,4 +2284,134 @@ describe('Component: UiGrid', () => {
             });
         });
     });
+
+    describe('Scenario: grid states', () => {
```

Comment:
nit: State alone doesn't seem to be the right word for what we're testing...
```suggestion
    describe('Scenario: Grid State Templates', () => {
```